lighthouse-extension/test/app/srclightrider-entry.js

```diff
@@ -6,12 +6,12 @@
 'use strict';
```

Comment:
I think you missed a `/` in the `git mv` :)

Reply:
oh nice thanks